[Docs]: Add documentation for installation on KinD (#1301)
layer5io/meshery: docs/pages/installation/kinD.md

chmod +x ./kind
mv ./kind /some-dir-in-your-PATH/kind
```
If you are running Ubuntu on WSL2, use `Docker Ubuntu` distro to install `Docker`. 

### Create cluster using KinD

> In order to let you successfully access Meshery server from your localhost, you need to follow the specific instructions

Reviewer: Hey @tingkai-zhang , just a small change I would suggest if you would incorporate.
```suggestion
In order to successfully build the meshery server on your local machine, you will need to follow these specific instructions
```

Author: Thanks @kushthedude I think that way makes it more clear to the readers.

according to your Operating System to complete the creation of KinD cluster.

#### KinD on WSL2

> First, we will get the ip address of your WSL2 distro by:

Reviewer: ```suggestion
First, we will get the IP address of your WSL2 distro by:
```

mv ./kind /some-dir-in-your-PATH/kind
```

Next we will demonstrate how to install Meshery with KinD on WSL2 

> ### Create cluster using KinD

Reviewer 2: Thank you for this, @tingkai-zhang. If the user is running KinD on Mac or Linux, they will need to execute steps inside of this "Create cluster using KinD" section, but only specific steps need to be executed in the event they are using WSL, right? Can we segregate the create cluster instructions based on OS?

Reviewer 3: Yep, @leecalcote was right. There may multiple situations `MacOS/Linux` and `WSL`(I'm not sure, I saw many of the articles includes `WSL2`) .So.please kindly confirm `WSL2` or `WSL` is more fit the primary developer. And thanks for your contributions.

Author: Thanks for the review. I have updated the instructions.

Reviewer 2: Thank you, @tingkai-zhang 👍
